### ISSUE TITLE
Keyboard Tab Through Radio Button Issue 40010

### ISSUE THREAD

**Issue by sdwjoomla** [joomla-projects/guided-tours#30]
The issue tracker link: https://github.com/joomla/joomla-cms/issues/40010

**Comment by garstud** [comment]
## Reason for feature
Accessibility issue, keyboard navigation fails to go back and forth on radio buttons

## More information
Not a bug, arrows works well!
tab and shift+tab works well to change the field selection.

**Comment by sdwjoomla** [comment]
@garstud @obuisard should this be closed?

**Comment by obuisard** [comment]
> @garstud @obuisard should this be closed?

No, we need to find a way to improve the flow for radio buttons
